Project/Source Files/Services/Networking/Helper.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Helper.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+struct Helper {
```

Comment:
`Helper` is generally not the best name as it doesn't tell specifically what it does and it can be anything. It's better to name it more specific.